src/tests/ftest/recovery/ddb.py

```diff
@@ -143,28 +157,16 @@ def test_recovery_ddb_ls(self):
             # /d4e0c836-17bd-4df3-b255-929732486bab/281479271677953.0.0/
             # [0] 'Sample dkey 0 0' (15)
             # [1] 'Sample dkey 0 1' (15)
-            match = re.findall(dkey_akey_regex, ls_out)
+            match = re.findall(dkey_regex, ls_out)
 
             actual_dkey_count += len(match)
 
             # Verify dkey string.
-            actual_dkey_1 = " ".join(match[0][1].split())
-            actual_dkey_2 = " ".join(match[1][1].split())
-            # We're not testing the numbers in the string because it's not deterministic.
-            if self.random_dkey not in actual_dkey_1:
-                msg = ("Unexpected dkey! obj_i = {}. Expected = {}; "
-                       "Actual = {}").format(obj_index, self.random_dkey, actual_dkey_1)
-            if self.random_dkey not in actual_dkey_2:
-                msg = ("Unexpected dkey! obj_i = {}. Expected = {}; "
-                       "Actual = {}").format(obj_index, self.random_dkey, actual_dkey_2)
-
-            # Verify the dkey size field.
-            for dkey in match:
-                dkey_string = dkey[1]
-                # int(dkey[2]) is the dkey size; avoid pylint error.
-                if len(dkey_string) != int(dkey[2]):
-                    msg = ("Wrong dkey size! obj_index = {}. String = {}; "
-                           "Size = {}").format(obj_index, dkey_string, int(dkey[2]))
+            for i in range(self.dkey_count):
```

Comment:
> GitHub Actions / Pylint check
>
> invalid-name, Variable name "i" doesn't conform to snake_case naming style

Variable names have to be at least 3 characters long
```suggestion
            for idx in range(self.dkey_count):
```